app/retail/templates/emails/quarterly_stats.html

```diff
@@ -0,0 +1,230 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize %}
+
+{% block content %}
+
+<style>
+  h1, h2, h3, h4 {
+    text-transform: none;
+    color: #0D0764;
+  }
+  h2 {
+    margin-top: 3em;
+  }
+  h4 {
+    color: #9B9B9B;
+  }
+  hr {
+    color: #E0E0E0;
+  }
+  .primary-content {
+    margin-left: 20%;
+    margin-right: 20%;
+    padding: 5em;
+  }
+  .social-img {
+    margin-left: 5px;
+    margin-right: 5px;
+  }
+  .total-transaction {
+    background-color: #D6FBEB;
+    color: #00A55E;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .stat-box {
+    display: flex;
+    margin-bottom: 2em;
+    border-style: solid;
+    border-width: 1px;
+    border-color: #e0e0e0;
+  }
+  .stat-img {
+    text-align: center;
+    margin: 1em;
+    border-width: 1px;
+    border-radius: 50%;
+    height: 100px;
+    width: 15%;
+  }
+  .stat-img > .content {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  .stat-contents {
+    color: #0D0764;
+    text-align: left;
+    width: 85%;
+    height: 100px;
+    align-items: center;
+    display: flex;
+    justify-content: left;
+  }
+  .stat-contents > p {
+    color: #0D0764;
+  }
+  .bounty-img {
+    border-radius: 50%;
+    background-color: #DBD0FF;
+    color: #3E00FF;
+    float: left;
+    width: 5em;
+    height: 5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .bounty-fund-img {
+    border-radius: 50%;
+    background-color: #D6FBEB;
+    color: #00A55E;
+    float: right;
+    width: 5em;
+    height: 5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: -3em;
+    margin-left: -1.3em;
+
+  }
+  .stat {
+    display: flex;
+    padding: 0.5em;
+    align-items: center;
+    justify-content: center;
+  }
+  #grow-oss {
+    margin-top: 5em;
+  }
+  .button-action {
+    margin-bottom: 5em;
+    margin-top: 3em;
+  }
```

Comment:
Mind leaving a line after each CSS class ? (we stick to that throughout the repo)